client/my-sites/activity/activity-log-v2/style.scss

```diff
@@ -9,29 +9,9 @@ main.activity-log-v2 {
 		}
 	}
 
-	.filterbar .filterbar__wrap.card {
-		display: flex;
-		flex-direction: row;
-
-		@include breakpoint-deprecated( "<660px" ) {
```

Comment:
Given these styles are generic for filterbar, decided to move it into `filterbar/style.scss` instead.